lib/charms/layer/kubernetes_common.py

```diff
@@ -637,36 +637,52 @@ def write_azure_snap_config(component):
 
 def configure_kube_proxy(configure_prefix, api_servers, cluster_cidr):
     kube_proxy_opts = {}
+    kube_proxy_config = {
```

Comment:
since 1.23, kube-proxy has been recommending we use a kube-proxy config file rather than configure with arguments